plugins/obs-webrtc/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+project(obs-webrtc)
+
+include(FetchContent)
+
+FetchContent_Declare(
+  Corrosion
+  GIT_REPOSITORY https://github.com/corrosion-rs/corrosion.git
+  GIT_TAG v0.3.1)
+
+FetchContent_MakeAvailable(Corrosion)
+
+add_library(obs-webrtc MODULE)
+
+if(OS_WINDOWS)
+  set(MODULE_DESCRIPTION "OBS webrtc module")
+  configure_file("${CMAKE_SOURCE_DIR}/cmake/bundle/windows/obs-module.rc.in"
+                 obs-webrtc.rc)
+  list(APPEND obs-webrtc_SOURCES obs-webrtc.rc)
```

Comment:
With the recent changes, this needs to use `target_sources()` instead. 

Also I usually put the generic sources _after_ the `add_library` call, then added conditional sources below that (makes it clearer which source files a library is made of), but I leave that up to you (it's just a stylistic choice, the outcome should be the same).

Reply:
Sounds good! I can make this change

Reply:
Moved stuff around a bit

Reply:
This will still not work - you add the file to `obs-webrtc_SOURCES` but you also do not use that variable to add sources to the target (you changed this after one of my earlier comments).

Just do `target_sources(obs-webrtc PRIVATE obs-webrtc.rc)` - it will append entries to the `SOURCES` property (it's a shorthand for `set_property(TARGET obs-webrtc APPEND PROPERTY SOURCES <YOUR_FILE>)`.